test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2523,8 +2523,8 @@ static public void testUnresolvedGetFieldUse() throws Throwable {
 								new UnresolvedClassDesc("UnresolvedB2", new String[] {"c:I", "d:I"}),
 								new UnresolvedClassDesc("UnresolvedB3", new String[] {"e:I", "f:I"})};
 
-		Class[] valueClassArr = new Class[uclassDescArr.length];
-		String containerFields[] = new String[uclassDescArr.length];
+		Class<?>[] valueClassArr = new Class[uclassDescArr.length];
```

Comment:
No raw types, please:
```java
		Class<?>[] valueClassArr = new Class<?>[uclassDescArr.length];
```
See also lines 2548, 2628 and 2656.

Reply:
Thanks I fixed all the `Class[]` cases.

Reply:
Thanks I fixed all the `Class[]` cases.